Project.toml

```diff
@@ -87,12 +88,13 @@ PrecompileTools = "1.1"
 Preferences = "1.3"
 Printf = "1"
 RecipesBase = "1.1"
+RecursiveArrayTools = "2.38.10"
 Reexport = "1.0"
 Requires = "1.1"
 SciMLBase = "1.90, 2"
 SimpleUnPack = "1.1"
 SparseArrays = "1"
-StartUpDG = "0.17.7"
+StartUpDG = "0.17.7, ~1.1.5"
```

Comment:
Are you sure about this setting? Why not allow minor version upgrades of StartUpDG (currently prevented by the `~` specifier). Also, why allow 0.17.7, 1.1.5, but not 1.0.0-1.1.4?

Reply:
No, I think I misunderstood the `~` specifier. I'll remove it; the carat should work fine. 

Versions 1.0.0-1.1.4 were all changes that failed to fix Trixi CI failures. I don't know a better way to do this than releasing small patch versions, since I can't run all CI tests locally.

Reply:
Removed `~` in https://github.com/trixi-framework/Trixi.jl/pull/2001/commits/551dec3a78cc00783b05f9a1493ca006758241d3